.github/workflows/ee_cli_release_fips.yml

```
@@ -0,0 +1,44 @@
+---
+name: release ee cli
+
+"on":
+  release:
+    branches:
```

actionlint (1.7.7)

"branches" filter is not available for release event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)